examples/data/meta.yaml

```diff
@@ -39,9 +39,9 @@ default:
     - 85 to 90
     - 90 to 125
 
-USA:
-  age_groups:
-    - 0 to 65
-    - 65 to 125
+# USA:
```

Comment:
Looks like we're still getting an error when locations are missing sex-specific data. Until this is fixed I'm not going to incorporate the sex-specific modeling into my own code.

Reply:
What error are you getting?

Reply:
> Looks like we're still getting an error when locations are missing sex-specific data. Until this is fixed I'm not going to incorporate the sex-specific modeling into my own code.

What error are you getting?

Reply:
I got this error until I removed locations that are only all sex.

```Traceback (most recent call last):
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 279, in <module>
data_age_mp = fit_age_mp(dmanager)
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 238, in fit_age_mp
results[location] = fit_age_mp_location(location, dmanager)
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 229, in fit_age_mp_location
models = get_model_mp(data_0,
File "/ihme/code/demographics/krpaul/model_covid_em//02b_fit_mscm_model.py", line 46, in get_model_mp
num_knots = max(2, int(knots_per_year*df.time.max()/units_per_year))
ValueError: cannot convert float NaN to integer
```

Reply:
I think it is supposed to cause errors for locations with only all sex data? Since if you want to have sex specific cascade you need to provide them first.